src/runtime/runtime_impl.rs

```diff
@@ -227,27 +206,13 @@ impl Runtime {
     ///
     /// [mod]: index.html
     pub fn new() -> io::Result<Self> {
-        // Spawn a reactor on a background thread.
-        let reactor = Reactor::new()?.background()?;
-
-        // Get a handle to the reactor.
-        let handle = reactor.handle().clone();
-
-        let pool = threadpool::Builder::new()
-            .name_prefix("tokio-runtime-worker-")
-            .around_worker(move |w, enter| {
-                ::tokio_reactor::with_default(&handle, enter, |_| {
-                    w.run();
-                });
-            })
-            .build();
-
-        Ok(Runtime {
-            inner: Some(Inner {
-                reactor,
-                pool,
-            }),
-        })
+        Builder::new()
+            .threadpool_builder(
+                threadpool::Builder::new()
+                    .name_prefix("tokio-runtime-worker-")
```

Comment:
This should be the default on `runtime::Builder::new`, so this function just becomes:

```rust
Builder::new().build()
```

Reply:
Done